python/ray/rllib/models/model.py

```diff
@@ -15,12 +15,15 @@ class Model(object):
 
     Attributes:
         inputs (Tensor): The input placeholder for this model.
+        prefix (str): Unique identifier to prevent name clashes in the
```

Comment:
Hm do you actually need this? I thought name scopes were hierarchical so you can reuse this across multiple distinct scopes.